pkgs/dart_model/lib/src/dart_model.g.dart

```diff
@@ -131,3 +141,45 @@ extension type QualifiedName.fromJson(String string) {
 extension type Query.fromJson(Map<String, Object?> node) {
   Query() : this.fromJson({});
 }
+
+enum StaticTypeType {
+  unknown,
+  nullableType,
+  voidType;
+}
+
+extension type StaticType.fromJson(Map<String, Object?> node) {
```

Comment:
It would be useful to see schema change + code change + generator together, but I don't see the schema change--could you add it please?

There isn't any other test coverage for the generator besides what is in the schema, so it probably makes sense to add something to the schema even if it's just showing the new functionality. In that case please note in the description that the type is currently just there to show the union functionality :)

Reply:
I didn't mean to commit the type changes in this PR, but I've added a few types with their schema definitions to test the generator.